changed path handling to be OS agnostic

MarcoKe opened the pull request:
## Types of changes

<!--- What types of changes does your code introduce? Put an `x` in all the boxes that apply: -->

- [ ] Docs change / refactoring / dependency upgrade
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Motivation and Context / Related issue

<!--- Why is this change required? What problem does it solve? -->
Since Windows uses backslashes instead of forwardslashes, replacing parts of a path based on forwardslashes resulted in an error under Windows.
<!--- Please link to an existing issue here if one exists. -->
[https://github.com/facebookresearch/mbrl-lib/issues/152](https://github.com/facebookresearch/mbrl-lib/issues/152)

## How Has This Been Tested (if it applies)

<!--- Please describe here how your modifications have been tested. -->
The changes were manually tested using Windows 10 and a VM with Ubuntu 20.04. 

The test suite was executed using pytest and no new errors were found. 

## Checklist

<!--- Go over all the following points, and put an `x` in all the boxes that apply. -->
<!--- If you're unsure about any of these, don't hesitate to ask. We're here to help! -->

- [x] The documentation is up-to-date with the changes I made.
- [x] I have read the [**CONTRIBUTING**](../../CONTRIBUTING.md) document and completed the CLA (see **CONTRIBUTING**).
- [x] All tests passed, and additional code has been covered with new tests. (Some tests already did not pass with a fresh clone of the main branch. Those still do not pass with the changes of course. 

<!--- In any case, don't hesitate to join and ask questions if you need on MBRL-Lib users Facebook group https://www.https://www.facebook.com/groups/mbrl.lib -->

luisenp commented on mbrl/diagnostics/training_browser.py:
```diff
@@ -50,10 +51,9 @@ def reloadExperiments(self, data):
         self.data = []
         for path in data:
             if path.endswith(SOURCE):
+                config_path = Path(path).parent / Path(".hydra/config.yaml")
```
nit: I think you don't need to also wrap ".hydra/config.yaml" into a `Path` object.

luisenp reviewed (approved):
LGTM, thanks for reporting and adding a quick fix!

luisenp commented:
Hi @MarcoKe. Can you rebase on top of the latest changes to `main`? I fixed some import issues that are causing CI to fail. After that, we are good to merge. Thanks!

MarcoKe commented:
> Hi @MarcoKe. Can you rebase on top of the latest changes to `main`? I fixed some import issues that are causing CI to fail. After that, we are good to merge. Thanks!

Done. I also removed the unnecessary wrap in a Path object.
